test/mock-utils.ts

```diff
@@ -0,0 +1,20 @@
+// NOTE: This file is referenced as a pathGroup pattern in .eslintrc (import/order)
+
+import fs from 'fs';
+
+// Declare originalFs outside of mockReadFileSync to prevent infinite recursion errors in mockReadFileSync.
+const originalFs = fs.readFileSync;
```

Comment:
TIL

Reply:
We may or may not need this for E2E tests, I'll remove it with that ask if not needed.